### PR TITLE
Add zoom-control extension to control zoom meetings

### PR DESCRIPTION
**Pull request by prashantv**
## Description

Add extension to control the current Zoom meeting using Applescript to send commands to the Zoom application. Initial support for:
 * mute/unmute
 * start/stop video

## Screencast

https://github.com/raycast/extensions/assets/140159/1c4884d6-f7c4-4d7c-8b55-087479d9cb14

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Seems like if Zoom is not running it shows both an error toast and a HUD

Raycast-nDv2Yrkg@2x

Could you look into this?

**Comment by prashantv** [comment]
Thanks for taking a look and for fixing up a few issues!

I've updated the extension to only show the error toast if zoom is not running, or there's no active meeting.

**Comment by pernielsentikaer** [comment]
Thanks for creating this and thank you for the collaboration. It was a pleasure working with you. 🙂

**Comment by pernielsentikaer** [comment]
One thing I noticed in the `README` is that you mention `Leaving a meeting` - is that something you'd like to implement now or later?

File: extensions/zoom-control/package.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  ],
+  "categories": [
+    "Communication"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "icon": "zoom-mute.png",
+      "name": "mute",
+      "title": "Zoom - Mute Meeting",
```
Comment (erics118) [inline extensions/zoom-control/package.json:19]: Should the `Zoom - ` part be removed from the command? The extension guidelines suggest it just being a `<verb> <noun>`, without a prefix

**Review by erics118 — CHANGES_REQUESTED** [review]

File: extensions/zoom-control/package.json (hunk 1 above)
Comment (pernielsentikaer) [reply extensions/zoom-control/package.json:19]: It should! I already changed that [here](https://github.com/raycast/extensions/pull/8817/commits/a6b96a0aeafbb70f0af41a9365cce860f9f11803) but it seems like it got pushed back 🙂

**Comment by prashantv** [comment]
> One thing I noticed in the `README` is that you mention `Leaving a meeting` - is that something you'd like to implement now or later?

Ah I was hoping to implement it in the first version, but there's no menu item for leaving a meeting, so I'll look into later. Updated the README to remove mention of it.

**Comment by prashantv** [comment]
Gentle ping -- anything else needed before this can be merged?

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by prashantv** [comment]
Linked account

**Pull request merged**